app/javascript/hyacinth_ui_v1/graphql/terms.js

```diff
@@ -0,0 +1,91 @@
+import { gql } from 'apollo-boost';
+
+export const getTermsQuery = gql`
+  query Terms($vocabularyStringKey: ID!, $limit: Limit!, $offset: Offset, $query: String) {
+    vocabulary(stringKey: $vocabularyStringKey) {
+      stringKey
+      label
+      locked
+      customFieldDefinitions {
+        fieldKey
+        label
+        dataType
+      }
+      terms(limit: $limit, offset: $offset, query: $query) {
+        totalCount
+        nodes {
+          id
+          uri
+          prefLabel
+          altLabels
+          authority
+          termType
+          customFields {
+            field
+            value
+          }
+        }
+      }
+    }
+  }
+`;
+
+
+export const getTermQuery = gql`
+  query Term($vocabularyStringKey: ID!, $uri: ID!) {
+    vocabulary(stringKey: $vocabularyStringKey) {
+      stringKey
+      label
+      locked
+      customFieldDefinitions {
+        fieldKey
+        label
+        dataType
+      }
+      term(uri: $uri) {
+        id
+        uri
+        prefLabel
+        altLabels
+        authority
+        termType
+        customFields {
+          field
+          value
+        }
+      }
+    }
+  }
+`;
+
+export const createTermMutation = gql`
+  mutation CreateTerm($input: CreateTermInput!) {
+    createTerm(input: $input) {
+      term {
+        id
+        uri
+        prefLabel
+      }
+    }
+  }
+`;
+
+export const updateTermMutation = gql`
+  mutation UpdateTerm($input: UpdateTermInput!) {
+    updateTerm(input: $input) {
+      term {
+        uri
```

Comment:
in the future, we should discuss whether we want to send the same term fields as show back and redraw after update.

Reply:
resolving since this is known to be future work

Reply:
Agreed, createTerm has more fields because I discovered that TermSelect depended on a more complete term being returned.